openupgradelib/openupgrade_merge_records.py

```diff
@@ -242,18 +242,17 @@ def _adjust_merged_values_orm(env, model_name, record_ids, target_record_id,
 
       Possible operations by field types:
 
-      * Char fields:
-        - 'merge': content is concatenated with an ' | ' as separator
-        - other value (default): content on target record is preserved
-      * Text and Html fields:
-        - 'merge' (default): content is concatenated with an ' | ' as separator
-        - other value: content on target record is preserved
+      * Char, Text and Html fields:
+        - 'merge' (default for Text and Html): content is concatenated
+           with an ' | ' as separator
+        - other value (default for Char): content on target record is preserved
       * Integer, Float and Monetary fields:
-        - 'sum' (default): Sum all the values of the records.
+        - 'sum' (default for Float and Monetary): Sum all records's values.
```

Comment:
Saxon genitive is for persons: https://escuelavictoria.es/saxon-genitive-2/, so you should use previous form.